gating/check/run_deploy_mnaio.sh

```diff
@@ -16,64 +16,47 @@
 
 ## Shell Opts ----------------------------------------------------------------
 
-set -evu
-
-echo "Building a Multi Node AIO (MNAIO)"
-echo "+-------------------- MNAIO ENV VARS --------------------+"
-env
-echo "+-------------------- MNAIO ENV VARS --------------------+"
-
-## Vars ----------------------------------------------------------------------
-
-# These vars are set by the CI environment, but are given defaults
-# here to cater for situations where someone is executing the test
-# outside of the CI environment.
-export RE_JOB_NAME="${RE_JOB_NAME:-}"
-export RE_JOB_IMAGE="${RE_JOB_IMAGE:-}"
-export RE_JOB_BRANCH="${RE_JOB_BRANCH:-master}"
-export RE_JOB_SCENARIO="${RE_JOB_SCENARIO:-swift}"
-export RE_JOB_ACTION="${RE_JOB_ACTION:-deploy}"
-export RE_JOB_FLAVOR="${RE_JOB_FLAVOR:-}"
-export RE_JOB_TRIGGER="${RE_JOB_TRIGGER:-PR}"
-export RE_HOOK_ARTIFACT_DIR="${RE_HOOK_ARTIFACT_DIR:-/tmp/artifacts}"
-export RE_HOOK_RESULT_DIR="${RE_HOOK_RESULT_DIR:-/tmp/results}"
-
-## Functions -----------------------------------------------------------------
+set -exu
+
+echo "Installing RPC-O on a Multi Node AIO (MNAIO)"
+
+## Vars and Functions --------------------------------------------------------
+
+source "$(readlink -f $(dirname ${0}))/../gating_vars.sh"
+
 source /opt/rpc-openstack/scripts/functions.sh
 
-## OSA MNAIO Vars
-export PARTITION_HOST="true"
-export NETWORK_BASE="172.29"
-export DNS_NAMESERVER="8.8.8.8"
-export OVERRIDE_SOURCES="true"
-export DEVICE_NAME="vda"
-export DEFAULT_NETWORK="eth0"
-export DEFAULT_IMAGE="ubuntu-16.04-amd64"
-export DEFAULT_KERNEL="linux-image-generic"
-export SETUP_HOST="true"
-export SETUP_VIRSH_NET="true"
-export VM_IMAGE_CREATE="true"
-export DEPLOY_OSA="true"
-export OSA_BRANCH="${OSA_RELEASE}"
-export PRE_CONFIG_OSA="true"
-export RUN_OSA="false"
-export CONFIGURE_OPENSTACK="false"
-export DATA_DISK_DEVICE="sdb"
-export CONFIG_PREROUTING="false"
-export OSA_PORTS="6080 6082 443 80 8443"
-export RPC_BRANCH="${RE_JOB_BRANCH}"
-export DEFAULT_MIRROR_HOSTNAME=mirror.rackspace.com
-export DEFAULT_MIRROR_DIR=/ubuntu
-export INFRA_VM_SERVER_RAM=16384
-export MNAIO_ANSIBLE_PARAMETERS="-e default_vm_disk_mode=file"
-export DEPLOY_MAAS=false
-# ssh command used to execute tests on infra1
-export MNAIO_SSH="ssh -ttt -oStrictHostKeyChecking=no root@infra1"
-# place variable in file to be sourced by parent calling script 'run'
-export MNAIO_VAR_FILE="${MNAIO_VAR_FILE:-/tmp/mnaio_vars}"
-echo "export MNAIO_SSH=\"${MNAIO_SSH}\"" > "${MNAIO_VAR_FILE}"
-echo "export RPC_RELEASE=\"${RPC_RELEASE}\"" >> "${MNAIO_VAR_FILE}"
-echo "export RPC_PRODUCT_RELEASE=\"${RPC_PRODUCT_RELEASE}\"" >> "${MNAIO_VAR_FILE}"
+source "$(readlink -f $(dirname ${0}))/../mnaio_vars.sh"
+
+# If there are images available, and this is not the 'deploy' action,
```

Comment:
is or is not?

Reply:
Ah yes, that should be ``is the deploy action``. I'll have to correct that in master and port it back.

Reply:
PR up: https://github.com/rcbops/rpc-openstack/pull/3308